src/main/java/teammates/storage/entity/FeedbackSession.java

```diff
@@ -282,7 +293,10 @@ public String toString() {
                 + ", sentPublishedEmail=" + sentPublishedEmail
                 + ", isOpeningEmailEnabled=" + isOpeningEmailEnabled
                 + ", isClosingEmailEnabled=" + isClosingEmailEnabled
-                + ", isPublishedEmailEnabled=" + isPublishedEmailEnabled + "]";
+                + ", isPublishedEmailEnabled=" + isPublishedEmailEnabled
+                + "]";
+        // todo should the tostring be changed to include sentOpeningEmail?
```

Comment:
yes I think we should add that

Reply:
Done, i've added openingSoon, closing and sentClosedEmail